wasmcloud_host/lib/wasmcloud_host_web/live/page_live.html.leex

```diff
@@ -135,77 +136,91 @@
                     </svg>&nbsp;
                   </button>
                 </td>
-              </tr>
-            <% end %>
-          </tbody>
-        </table>
-      </div>
+                <!-- <td style="text-align: center">
```

Comment:
Can we remove the commented stuff?

Reply:
Yeah that was leftover from some prototyping, I'll get it cleaned up